tests/tests.qbs

```diff
@@ -0,0 +1,100 @@
+import qbs
+import qbs.FileInfo
+import qbs.File
+
+Project {
+    name: "QSqlMigrator.tests"
+
+
+    Application {
+        name: "ApiTest"
+        consoleApplication: true
+        type: base.concat("autotest")
+        install: true
+
+        Depends { name: "QSqlMigrator" }
+        Depends { name: "cpp" }
+        Depends { name: "Qt"; submodules: ["core", "sql", "test"] }
+        Depends { name: "SqliteMigrator" }
```

Comment:
"SqliteMigrator" should provide "QSqlProvider", "cpp" and "core"+"sql" through exports already.